pkgs/os-specific/linux/oxtools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub
+, glibc, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "0xtools";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "tanelpoder";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-pe64st3yhVfZi8/sTEfH1cNjx7JpqxDmxMmodpXnqaU=";
+  };
+
+  postPatch = ''
+    substituteInPlace lib/0xtools/proc.py \
+      --replace /usr/include/asm/unistd_64.h ${glibc.dev}/include/asm/unistd_64.h
+  '';
+
+  propagatedBuildInputs = [ python3 ];
```

Comment:
Why do we propagate python3? Is it required at runtime? If so I'd rather would like to wrap it since propagate should be used carefully outside of pythonPackages.